docs/operator-bundle.md

```diff
@@ -50,10 +50,21 @@ metadata:
 EOF
 ```
 
-Deploy the operator using the operator-sdk:
+### Download operator-sdk
+
+Follow instructions [here](https://sdk.operatorframework.io/docs/installation/#install-from-github-release).
```

Comment:
Do we want to mention alternative way: `make operator-sdk`?

Reply:
Good idea. Updating.